### DISCUSSION TITLE
xarray/dask: count of tasks and dask graph

### DISCUSSION THREAD

**Discussion by fhhan** [pydata/xarray#9837] [Q&A] [answered]
Hi everyone, I have a question about  the show info of tasks in jupyter-lab.

when I read this article [Xarray Anomaly Calculations](https://gist.github.com/rabernat/30e7b747f0e3583b5b776e4093266114), I think it's great to know the count of tasks. With this information, I can learn different groupby strategies in [flox🔥](https://flox.readthedocs.io/en/latest/implementation.html) and choose the best method.
![PixPin_2024-11-28_11-07-15](https://github.com/user-attachments/assets/86ef7ee3-31e9-4c58-89c7-89b7e638917d)

However in my local jupyter-lab, the info of xarray data just like:
![PixPin_2024-11-27_20-09-44](https://github.com/user-attachments/assets/1dd52168-c57b-4eb4-b6a9-86d61710a601)

So, how to show the number of tasks in jupyter-lab?  
Or any relationship between tasks and dask graph, so we can deduce the tasks from dask graph?

Thank you all~

**Comment by dcherian** [comment]
> groupby strategies in [flox🔥](https://flox.readthedocs.io/en/latest/implementation.html) and choose the best method.

flox will do this automatically now, with no user input needed. 


Are you running in to trouble with a calculation?

**Reply by fhhan** [reply]
thank you, dcherian. flox is so great!

I’m trying to determine the count of tasks using different methods, but I’m unsure if the issue lies with Jupyter or xarray. The output doesn’t provide any information about the number of tasks (as shown in two pictures). Could you help clarify this?

**Reply by dcherian** [reply, marked as answer]
dask has removed this information from the repr. Why do you need it?

**Reply by fhhan** [reply]
OK. I think knowing the number of tasks can provide a more intuitive understanding of the complexity of the tasks.Or is my perspective incorrect?
